charts/opentelemetry-kube-stack/templates/_config.tpl

```diff
@@ -41,7 +41,7 @@ the config is written as YAML.
 {{- $config = (include "opentelemetry-kube-stack.collector.applyClusterMetricsConfig" (dict "collector" $collector "namespace" .namespace) | fromYaml) -}}
 {{- $_ := set $collector "config" $config }}
 {{- end }}
-{{- toYaml $collector.config | nindent 4 }}
+{{- tpl (toYaml $collector.config) . | nindent 4 }}
```

Comment:
the change here makes sense, but it would be useful to see this working with a modified example. It would be easy enough to template in something like an annotation to check

Reply:
oh i must have missed that! nope youre all good. The examples tend to be blend together during review. Thank you :bow: